doc/readme/EXAMPLES.md

```diff
@@ -94,6 +94,16 @@ You can render it with
 which is also part of `make doc`.
 The target might not be available if the requirements in `doc/cookbook/requirements.txt` are not satisfied (in particular Sphinx), or if the meta examples are disabled.
 
+If the following error is still visible
+
+    make: *** No rule to make target 'cookbook'. Stop.
+
+Find out if value of `SPHINX_EXECUTABLE` is set by searching for it in CMakeCache.txt which is present in the build directory. If it's not set then not all requirements in `doc/cookbook/requirements.txt` have been satisfied. 
```

Comment:
in this case it is the sphinx-build that has not been found...

Reply:
Added.